### ISSUE TITLE
Regarding DTX files defining separate ride/crash + open/close hi-hat and conversion to pro drums

### ISSUE THREAD

**Issue by sakamoto-neko** [mtolly/onyxite-customs#242]
A few DTX simfile charters do define separate notes for ride/crash cymbals and open/close hi-hat, which can be viewed if you opened them up in DTXCreator NX. Would it be possible to generate a conversion to pro drums (following official charting standards)? I can provide a few DTX samples if needed.

**Comment by mtolly** [comment]
Yeah, it would be good to add some preferences to control which RB cymbal each DTX cymbal maps to by default.

One option you could try now is, I and some others have been working on a "mostly DTX" MIDI-file drum format for the YARG project (still in design/planning phase currently). If you use the current Onyx build to convert from DTX to Clone Hero, it will include this track in the MIDI as it exists now (named `PART TRUE_DRUMS`), so if you can use Reaper or another sequencer then you can access the separate drum lanes and move them however you want. However it may not be as helpful if you want to treat open/closed hihat separately since they are encoded as a single hihat note + an open/closed modifier on top.

Reaper resources for the track if they are useful:
[Note names](https://github.com/mtolly/onyxite-customs/files/11688652/true-drum-notes.txt)
Color map
![true-drums](https://github.com/mtolly/onyxite-customs/assets/1427698/fee2ad64-9835-485a-8913-8d86716cf89b)
